### ISSUE TITLE
Insecure TLS support?

### ISSUE THREAD

**Issue by tranhl** [davesters/rn-native-mqtt#2]
@davesters As per title, just wondering if there was insecure TLS support, and if so, how do we go about configuring it?

**Comment by davesters** [comment]
I don't think it supports that right now, but it should not be too difficult to add. I can take a crack at it in the next day or two. And you are also welcome to submit a pull request. Just need to expose an option in the JS interface file and then map that to the right setting for both iOS and Android native MQTT client libraries.
